rfc/0003-policy-signing.md

```diff
@@ -0,0 +1,440 @@
+|              |                                  |
+| :----------- | :------------------------------- |
+| Feature Name | Policy Signing                   |
+| Start Date   | 21 Jan 2022                      |
+| Category     | Security                         |
+| RFC PR       | [#147](https://github.com/kubewarden/kubewarden-controller/pull/147)  |
+
+# Summary
+[summary]: #summary
+
+> Brief (one-paragraph) explanation of the feature.
+
+We want to provide a way to verify the integrity and authenticity of our policies, so that end-users can trust them.
```

Comment:
This RFC should mention https://slsa.dev as the 'directing force' behind our attempts to improve security of the software supply chain.

Reply:
Good point, added to the next iteration